apps/frontend/src/app/5_pages/RewardsPage/RewardsPage.tsx

```diff
@@ -89,9 +93,59 @@ const RewardsPage: FC = () => {
         <title>{t(translations.rewardPage.meta.title)}</title>
       </Helmet>
 
-      <div className="flex flex-col lg:flex-row gap-2 lg:gap-20 w-full lg:w-5/6 text-gray-10 mt-6 lg:mt-12">
-        <div className="w-full lg:w-80">
+      <div className="flex flex-col lg:flex-row gap-2 lg:gap-16 w-full lg:w-5/6 text-gray-10 mt-6 lg:mt-12">
+        <div className="flex flex-col gap-4 w-full lg:w-96">
           <TotalRewardsEarned />
+
+          <Banner
```

Comment:
The banners should have `Learn more` links.

Design:
<img width="507" alt="image" src="https://github.com/DistributedCollective/sovryn-dapp/assets/14878713/49f71aa1-a8fa-4179-9fb0-e942dcf7de71">

Implementation:
<img width="444" alt="image" src="https://github.com/DistributedCollective/sovryn-dapp/assets/14878713/626c21f6-e76e-4581-9b2e-e7f71a687e41">